src/features/user/RegisterTrees/RegisterTrees/UploadImages.tsx

```diff
@@ -65,7 +53,19 @@ export default function UploadImages({
     }
   };
 
-  const { getRootProps, getInputProps, isDragActive } = useDropzone({
+  const onDrop = React.useCallback((acceptedFiles: File[]) => {
+    acceptedFiles.forEach((file) => {
+      const reader = new FileReader();
+      reader.readAsDataURL(file);
+      reader.onabort = () => console.log('file reading was aborted');
+      reader.onerror = () => console.log('file reading has failed');
+      reader.onload = (event) => {
+        uploadPhotos(event.target?.result);
+      };
+    });
+  }, []);
```

Comment:
There are a couple of unexpected `any` warnings here related to images. Can these be resolved? You may get some new warnings once this is resolved as well.